src/main/java/com/hms/flexyosisoftconnector/package.html

```diff
@@ -2,7 +2,7 @@
 <BODY>
 Ewon Flexy OSIsoft Connector.
 
-@version 3.0.10
+@version 3.0.11

```

Comment:
Please update the `@version` tag in all `package.html` files. For a past RELEASE commit example, see https://github.com/hms-networks/flexy-osisoft-connector/commit/04c31e35b327d05284ec161a2f205e625bb46785

Alternatively, you could add a commit to delete the `@version` tag from all *except* this (the main) `package.html` file. Then, there would only this file to update.

In newer connectors, we have opted to only include the `@version` tag in the main/root `package.html` file, leaving it out or replacing it with an `@since` tag if the package was added after initial connector release.